infra/modules/azure_app_service_exposed/locals.tf

```diff
@@ -9,6 +9,7 @@ locals {
     zone_balancing_enabled = local.tier != "s" && local.tier != "xs"
     is_slot_enabled        = local.tier == "s" || local.tier == "xs" ? 0 : 1
     always_on              = local.tier == "xs" ? false : true
+    command_line           = var.pm2_startup_file_name == null ? null : "pm2 start ${var.pm2_startup_file_name} -i max --no-daemon"
```

Comment:
**question**: do you think this way will also work with NextJS apps?

We have some NextJS apps running on App Service that are currently started with `yarn start`, which is defined as follow: `"start": "./node_modules/next/dist/bin/next start"`

PS: I found [this discussion](https://github.com/vercel/next.js/discussions/31461) that may be useful